### ISSUE TITLE
Add Variadic Parameters Support

### ISSUE THREAD

**Issue by kylejgabriel** [uber/mockolo#51]
Does not currently support Variadic Parameters in function signatures.

Protocol definition:
`func example(strings: String...)`

Expected mock:
`var exampleHandler: ((_ strings: [String]) -> ())?`

Actual mock:
`var exampleHandler: ((_ strings: String) -> ())?`

**Comment by trsxxii** [comment]
Hi there!

I see this issue has been closed, but it seems variadic parameters are still not supported. 

I found test code related to variadic parameters (the testVariadicFuncs method in [FuncBasicTests.swift](https://github.com/uber/mockolo/blob/1d1bcdccd0ef4436c1125322dd8ce850289b4fe6/Tests/TestFuncs/TestBasicFuncs/FuncBasicTests.swift#L12-L15)), but it's commented out.

Are there any plans to support this feature in the future?

Thanks!
